config/search/config.json

```diff
@@ -363,6 +363,155 @@
           }
         }
       }
+    },
+    {
+      "type": "newsletter-infos",
+      "on_path": "newsletter-infos",
+      "rdf_type": "http://data.vlaanderen.be/ns/besluitvorming#NieuwsbriefInfo",
+      "properties": {
+        "richtext": "http://mu.semte.ch/vocabularies/ext/htmlInhoud",
+        "subtitle": "http://dbpedia.org/ontology/subtitle",
+        "title": "http://purl.org/dc/terms/title",
+        "publicationDate": [
+          "^http://www.w3.org/ns/prov#generated",
+          "http://data.vlaanderen.be/ns/besluitvorming#heeftOnderwerp",
+          "^http://purl.org/dc/terms/hasPart",
+          "http://data.vlaanderen.be/ns/besluitvorming#isAgendaVoor",
+          "http://data.vlaanderen.be/ns/besluit#geplandeStart"],
+        "decisionStatusId": [
+          "^http://www.w3.org/ns/prov#generated",
+          "http://data.vlaanderen.be/ns/besluitvorming#resultaat",
+          "http://mu.semte.ch/vocabularies/core/uuid"
+        ],
+        "decisionStatusLabel": [
```

Comment:
Related to above, I propose to rename `decisionStatusLabel` to `decisionResultCodeLabel`

Reply:
renamed